lib/enums/query/props/server_properties.py

```diff
@@ -27,7 +27,7 @@ def _get_aliases():
         A method that returns all valid string alias mappings
         """
         return {
-            ServerProperties.HYPERVISOR_ID: ["host_id", "hv_id"],
+            ServerProperties.HYPERVISOR_NAME: ["hv_name"],
```

Comment:
I know you changed this because we discovered `host_id` wasn't reliable and didn't correlate with actual hypervisor ids -  make a comment about that here for others

Reply:
You will also need to change the docs